content/v2.0/reference/flux/stdlib/pagerduty/endpoint.md

```diff
@@ -62,14 +63,33 @@ _For more information, see [`pagerduty.sendEvent()`](/v2.0/reference/flux/stdlib
 
 ##### Send critical statuses to a PagerDuty endpoint
 ```js
-import "monitor"
 import "pagerduty"
+import "influxdata/influxdb/secrets"
 
-endpoint = pagerduty.endpoint(token: "mySuPerSecRetTokEn")
+token = secrets.get(key: "PAGERDUTY_TOKEN")
+e = pagerduty.endpoint(token: token)
 
-from(bucket: "example-bucket")
+crit_statuses = from(bucket: "example-bucket")
   |> range(start: -1m)
   |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
-  |> map(fn: (r) => { return {r with status: r._status} })
-  |> monitor.notify(endpoint: endpoint)
+
+crit_statuses
+  |> e(mapFn: (r) => {
+      obj = mapFn(r: r)      
+      return {r with _sent: string(v: 2 == (sendEvent(pagerdutyURL: url,
+        routingKey: obj.routingKey,
+        client: obj.client,
+        clientURL: obj.clientURL,
+        dedupKey: r._pagerdutyDedupKey,
+        class: obj.class,
+        eventAction: obj.eventAction,
+        group: obj.group,
+        severity: obj.severity,
+        component: obj.component,
+        source: obj.source,
+        summary: obj.summary,
+        timestamp: obj.timestamp,
+      ) / 100))}
+    })
+  })()
```

Comment:
I think this is from the pagerduty package.

The correct could would not include 

```
obj = mapFn(r: r)      
      return {r with _sent: string(v: 2 == (sendEvent(pagerdutyURL: url,
```

but something like (from docs: `// The `mapFn` parameter must be a function that returns an object with `routingKey`, `client`, `client_url`, `class`, `group`, `severity`, `eventAction`, `component`, `source`, `summary`, and `timestamp` as defined in the sendEvent function.
`:

```
crit_statuses
  |> e(mapFn: (r) => ({ r with
        routingKey: r.routingKey,
        client: r.client,
        clientURL: r.clientURL,
        class: r.class,
        eventAction: r.eventAction,
        group: r.group,
        severity: r.severity,
        component: r.component,
        source: r.source,
        summary: r.summary,
        timestamp: r._time,
      })
 )()
```

Maybe the values could be hardcoded by the user, but that's the sense 👍

Reply:
This holds for also for slack and pushbullet.